src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -0,0 +1,357 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FormEvent, useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import ShowHide from 'src/shared/components/show-hide/ShowHide';
+import Checkbox from 'src/shared/components/checkbox/Checkbox';
+import RadioGroup from 'src/shared/components/radio-group/RadioGroup';
+
+import blastSettingsConfig from './blastSettingsConfig.json';
+
+import {
+  getSelectedSequenceType,
+  getSelectedBlastProgram,
+  getSelectedSearchSensitivity,
+  getBlastSearchParameters
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSelectors';
+import {
+  setSequenceType,
+  setBlastDatabase,
+  setBlastProgram,
+  changeSensitivityPresets,
+  setBlastParameter
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
+
+import type {
+  SequenceType,
+  BlastProgram,
+  BlastSelectSetting,
+  BlastBooleanSetting,
+  BlastParameterName,
+  BlastSettingsConfig
+} from 'src/content/app/tools/blast/types/blastSettings';
+
+import styles from './BlastSettings.scss';
+
+const getParameterData = (name: BlastParameterName) => {
+  return (blastSettingsConfig as unknown as BlastSettingsConfig).parameters[
```

Comment:
Can we do something like the following so that we don't have to repeatedly do the casting?

```
import blastSettings from './blastSettingsConfig.json';
const  blastSettingsConfig = blastSettings as unknown as BlastSettingsConfig;
```

Reply:
✅